apps/api/src/app/storage/storage.controller.ts

```diff
@@ -18,44 +17,26 @@ import { GetUserProfilePictureSignedUrlCommand } from './usecases/get-user-profi
 @UseGuards(UserAuthGuard)
 @ApiExcludeController()
 export class StorageController {
-  constructor(
-    private getSignedUrlUsecase: GetSignedUrl,
-    private getUserProfilePictureSignedUrlUsecase: GetUserProfilePictureSignedUrl
-  ) {}
+  constructor(private getSignedUrlUsecase: GetSignedUrl) {}
 
   @Get('/upload-url')
   @ApiOperation({
     summary: 'Get upload url',
   })
   @ApiResponse(UploadUrlResponse)
   @ExternalApiAccessible()
-  async signedUrl(@UserSession() user: IJwtPayload, @Query('extension') extension: string): Promise<UploadUrlResponse> {
-    return await this.getSignedUrlUsecase.execute(
-      GetSignedUrlCommand.create({
-        environmentId: user.environmentId,
-        organizationId: user.organizationId,
-        userId: user._id,
-        extension,
-      })
-    );
-  }
-
-  @Get('/upload-url/profile')
```

Comment:
We don't need a dedicated separate endpoint, I've just extended the existing one `GET /upload-url`